editor/spatial_editor_gizmos.cpp

```diff
@@ -569,20 +569,13 @@ bool EditorSpatialGizmo::intersect_ray(Camera *p_camera, const Point2 &p_point,
 
 		Point2 center = p_camera->unproject_position(t.origin);
 
-		Transform orig_camera_transform = p_camera->get_camera_transform();
-
-		if (orig_camera_transform.origin.distance_squared_to(t.origin) > 0.01) {
```

Comment:
The `look_at` call is there for a reason, not having it cuses errors when selecting icons that are on the border of the viewport. I think a better solution would be to add another guard to this if statement:
`ABS(orig_camera_transform.basis.get_axis(2).dot(Vector3(0,1,0))) < 0.99`

So `look_at` won't be called if the camera is already aligned with the target icon.

Reply:
I was wondering what that code was doing.

Reply:
Thanks.